CHSHGame/Tests.qs

```diff
@@ -38,16 +38,16 @@ namespace Quantum.Kata.CHSHGame {
                 set wins = wins + 1;
             }
         }
-        Message($"Win rate {ToDouble(wins) / 1000.}");
+        Message($"Win rate {IntAsDouble(wins) / 1000.}");
 
-        AssertBoolEqual(wins >= 700, true,
+        EqualityFactB(wins >= 700, true,
```

Comment:
If you don't mind, there are currently plenty of EqualityFactB (because that was the renaming suggestions for hte existing code) everywhere, so let's address that as part of a separate pass over all katas to incorporate features of the standard libraries when appropriate.